.eslintrc.js

```diff
@@ -10,38 +10,4 @@ module.exports = {
     'no-useless-escape': 'off',
     'no-case-declarations': 'off',
   },
-  overrides: [
```

Comment:
This override and `templates/demo-store/.eslintrc.js` are causing some issue where the VS Code eslint suggestion and the one in CI are giving different suggestions.

There is probably something to fix, but do we really need this override?